test/livebook/runtime/evaluator_test.exs

```diff
@@ -1300,6 +1300,53 @@ defmodule Livebook.Runtime.EvaluatorTest do
       assert_receive {:runtime_evaluation_response, :code_1, terminal_text("6"), metadata()}
     end
 
+    test "evaluate erlang-module code", %{evaluator: evaluator} do
+      Evaluator.evaluate_code(
+        evaluator,
+        :erlang,
+        "-module(tryme). -export([go/0]). go() ->{ok,went}.",
+        :code_4,
+        []
+      )
+
+      assert_receive {:runtime_evaluation_response, :code_4,
```

Comment:
@fnchooft Again, I'm not very faimiliar with the test logic in Elixir, but I noticed that for most tests it actually returns a map, not a tuple. i changed this lines to 

```elixir
assert_receive {:runtime_evaluation_response, :code_4,
                      %{:text => "\"erlang module successfully compiled\""}, metadata()}
```